cdap-watchdog/src/main/java/co/cask/cdap/logging/logbuffer/ConcurrentLogBufferWriter.java

```diff
@@ -67,10 +67,11 @@
   private final AtomicBoolean closed;
 
   public ConcurrentLogBufferWriter(CConfiguration cConf,
-                                   List<LogBufferProcessorPipeline> pipelines) throws IOException {
+                                   List<LogBufferProcessorPipeline> pipelines, Runnable runnable) throws IOException {
```

Comment:
`runnable` is a very generic name in this context, call it `cleaner` like you do in the other file